src/main/java/io/openliberty/tools/intellij/util/LibertyGradleUtil.java

```diff
@@ -153,4 +158,43 @@ private static boolean containerVersion(String plugin) {
         return false;
     }
 
+    public static String getGradleSettingsCmd(Project project) {
+        GradleProjectSettings gradleProjectSettings = GradleSettings.getInstance(project).getLinkedProjectSettings(project.getBasePath());
+
+        if (gradleProjectSettings.getDistributionType().isWrapped()) {
+            // a wrapper will be used
+            String wrapperPath = getLocalGradleWrapperPath(project);
+            if (wrapperPath != null) {
+                return wrapperPath;
+            }
+        }
+        else if (DistributionType.LOCAL.equals(gradleProjectSettings.getDistributionType())) {
+            // local gradle to be used
+            String gradleHome = gradleProjectSettings.getGradleHome(); //it is null when the path to gradle is invalid
+            if (gradleHome != null) {
+                String gradlePath = getCustomGradlePath(gradleHome);
+                if (gradlePath != null) {
+                    return gradlePath;
+                }
+            }
+        }
+        return "gradle"; // default gradle
+    }
+
+    private static String getLocalGradleWrapperPath(Project project) {
+        String gradlew = SystemInfo.isWindows ? "gradlew.bat" : "./gradlew";
```

Comment:
Make sure to add the `./` for Windows' gradlew.bat. 

Ran into this issue testing: 
![image](https://user-images.githubusercontent.com/689163/202291232-c4b53a1c-b6f8-4359-b7f3-8ec6bc2e3aad.png)

Reply:
Do you mean `.\`, right? Or `./` also works on Windows?